docs/DeveloperGuide.adoc

```diff
@@ -804,11 +815,29 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |delete a person |remove entries that I no longer need
 
+|`* * *` |As a couple (student) user |Add/Delete/Compare timetable with my partner |keep track of my partner's free time
+
+|`* * *` |As a couple (student) user |create a Journal to Add/View/Delete |keep track on life couple event
+
+|`* * *` |As a new student user |search for campus shortcuts |avoid getting lost or drenched from rain
+
+|`* * *` |As a couple (student) user | |I can calculate my diet of how many calories i have burnt
+
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
 |`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
 
 |`*` |user with many persons in the address book |sort persons by name |locate a person easily
+
+|`*`| As a couple (student) user |receive birthday notifications |remember my partner's birthday
+
+|`*`| As a couple (student) user |view my calendar |take note of upcoming events
+
+|`*`| As a couple (student) user |I can view my journal |
```

Comment:
One star means they might not able to view their journal in the early version. Are you sure?